pyproject.toml

```diff
@@ -123,6 +123,8 @@ container = [
   "strawberry-graphql[opentelemetry]==0.236.0",  # need to pin version because we're monkey-patching
   "uvloop; platform_system != 'Windows'",
 ]
+test = [
+]
```

Comment:
Still need this extra?

Reply:
just to suppress the warning if we move to tox